libs/eo/metering-points/shell/src/lib/eo-metering-point-table.component.ts

```diff
@@ -135,7 +149,11 @@ import { EoBetaMessageComponent } from '@energinet-datahub/eo/shared/atomic-desi
       </ng-container>
 
       <mat-header-row *matHeaderRowDef="displayedColumns"></mat-header-row>
-      <mat-row *matRowDef="let row; columns: displayedColumns"></mat-row>
+      <mat-row
+        *matRowDef="let row; columns: displayedColumns"
+        (mouseenter)="onRowHover(row.gsrn)"
```

Comment:
Har Pernille sagt god for dette?

Reply:
Nej, men Søren har

Reply:
Det ser funky ud, må jeg indrømme. Helt korrekt vil jeg mene, Pernille skal ind over, men hun er jo en begrænset resource, så jeg ved ikke.. Det bestemmer du :)